test/test-async-functions.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const {assert} = require("chai");
+const sinon = require("sinon");
+
+const {setupTestDOMWindow} = require("./setup");
+
+describe("browser-polyfill", () => {
+  describe("wrapped async functions", () => {
+    it("returns a promise which resolves to the callback parameters", () => {
+      const fakeChrome = {
+        alarms: {clear: sinon.stub()},
+        runtime: {lastError: undefined},
+      };
+      return setupTestDOMWindow(fakeChrome).then(window => {
+        fakeChrome.alarms.clear
+          .onFirstCall().callsArgWith(1, "res1")
+          .onSecondCall().callsArgWith(1, "res1", "res2", "res3");
+
+        return Promise.all([
+          window.browser.alarms.clear("test1"),
+          window.browser.alarms.clear("test2"),
+        ]);
+      }).then(results => {
+        assert.equal(results[0], "res1", "The first call resolved to a single value");
```

Comment:
@kmaglione I didn't applied this change yet, mostly because I want to be sure that I didn't misunderstood it, do you mean something like:

```
// in every test file:
const {equal, deepEqual} = require("chai").assert;
```

or something like:

```
// in the setup.rb
global.assert = require("chai").assert;

// then in every test file
const {equal, deepEqual} = assert;
```